src/stories/Autocomplete/Component/AdvancedParameters.stories.tsx

```diff
@@ -97,3 +98,21 @@ addComponentStoryDescription(
   `const args = ${stringifyWithDefaults(Debounce.args)}`,
   debounceDescription
 );
+
+export const Translations = ComponentTemplate.bind({});
+Translations.args = {
+  apiKey,
+  onSubmit,
+  advancedParameters: {
+    numTermsWithGroupSuggestions: 1,
+    numGroupsSuggestedPerTerm: 2,
+    translations: {
+      in: 'inside',
```

Comment:
Can we use a different language in our example so people understand the use case better?